packages/block-editor/src/components/block-compare/block-view.js

```diff
@@ -31,8 +31,7 @@ export default function BlockView( {
 
 			<div className="block-editor-block-compare__action">
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen when any block as a validation error and you click on resolve.

| Before | After     |
| :-------- | :------- |
| ![block compare 36](https://github.com/user-attachments/assets/d9d60d40-41ce-40f8-b569-2bac501be569)| ![block compare 40px ](https://github.com/user-attachments/assets/631dd8af-093e-40f6-87bb-e834270bb508) |